docs/pages/connect-your-client/teleport-connect.mdx

```diff
@@ -33,14 +33,26 @@ $ tar -xf  teleport-connect-(=teleport.version=)-linux-*.tar.gz
 Download and run the installer `.exe` file. It will install and open Teleport Connect without
 further user input.
 
+The installer requires Administrator privileges and if you run the installer as
+a regular user it will automatically prompt for admin rights.
+If you run the installer as an administrator (start cmd.exe with "Run as administrator")
+then the prompt will not be necessary.
```

Comment:
I feel like we can drop this, anyone who needs to know this (system administrators) probably know this already. Regular users probably don't need this info.